tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/document/documents.controller.js

```diff
@@ -1426,7 +1455,7 @@
         vm.sortby = "desc";
         vm.sortexp = "documentName";
         vm.showSortExp = function () {
-            if (vm.sortexp != "" || vm.sortexp != undefined || vm.sortby != "" || vm.sortby != undefined) {
+            if ((vm.sortexp != "" || vm.sortexp != undefined) && (vm.sortby != "" || vm.sortby != undefined)) {

```

Comment:
remove these conditions and put validation for element. if element is there then apply css.